### DISCUSSION TITLE
Maintenance policy - vulnerabilities

### DISCUSSION THREAD

**Discussion by deltas1x** [netbox-community/netbox#11443] [General]
Hi,

If a vulnerability is found which versions would be covered via patching 3.4, 3.3 , 3.2 ? 

Thanks

**Comment by kkthxbye-code** [comment]
We do not maintain anything but current minor release, so if a vulnerability is found in 3.4.2, it would be addressed in e.g. 3.4.3 and not backported to any older releases.

**Reply by deltas1x** [reply]
Perfect thanks
